modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/OpenAPIResourceTest.java

```diff
@@ -447,6 +464,12 @@ private void _testGetOpenAPI(
 			null, objectDefinition1.getRESTContextPath() + "/openapi.json",
 			Http.Method.GET);
 
+		_assertOpenAPIYAML(
```

Comment:
Move this after all the assertions for the json version. Now it is in the middle and it interrupts the expected flow of the test.